src/probe.ts

```diff
@@ -66,6 +78,13 @@ export interface HttpGetProbeOptions extends ProbeOptions {
    * @default - defaults to `container.port`.
    */
   readonly port?: number;
+
+  /**
+   *  Scheme to use for connecting to the host (HTTP or HTTPS).
+   *
+   *  @default - Defaults to HTTP.
```

Comment:
Where possible, and if they sufficiently convey meaning, we use concrete values to describe defaults.
 
```suggestion
   *  @default HttpGetScheme.HTTP.
```